server/src/Diff.ts

```diff
@@ -1,7 +1,34 @@
 import { diffWords } from 'diff';
 import * as fastDiff from 'fast-diff';
+import { Position } from 'vscode-languageserver-textdocument';
 
 export class Diff {
+  static convertPositionStraight(first: string, second: string, positionInFirst: Position): Position | undefined {
```

Comment:
What doest it mean to get converted position is undefined? May be add comment about return values?